### PR TITLE
Create Solitairestation.dmm

### PR DESCRIPTION
**Pull request by TheOnlySolitaire**
## About The Pull Request

This would add my map, Solitaire station, to the rotation for potential testing and feedback to improve on it. 
It is a completely new map addition.

Engineering Main:
SolitaireStation - engineering main

Cargo, comms, tech storage, library and chapel:
SolitaireStation - comms tech storage cargo library and chapel

Service:
SolitaireStation - service main

science main, arrivals and dorms:
SolitaireStation - science main, arrivals and dorms

Science annex:
SolitaireStation - science annex

Rec room, vault, perma and maints:
SolitaireStation - rec rooms, maints and perma

Security:
SolitaireStation - sec

AI sat and access:
SolitaireStation - ai sat

bridge, maints and virology:
SolitaireStation -bridge and virology

Medbay main:
SolitaireStation -medbay and evac

Eva, Aux base construction, public garden and maints:
SolitaireStation - eva, aux base, public garden and starboard fore maints

## Why It's Good For The Game

New maps are fun to play on and bring variety. I've tried to make it beginner friendly while also having enough there for more veteran players to find fun.

## Changelog
:cl:
add: SolitaireStation
/:cl:

### CODE REVIEW

**Comment by SpaceDragon00** [comment]
So Im going to compile 2 lists of issues. The first list is something I personally feel like you'll need to fix, and the second is just notes for Tom since he's the main maintainer and I'm not an experienced map reviewer.

### The Problems

- There are eleven stasis beds on total in this map. Thats... A lot. Like, a hell of a lot. It'll definitely need to be toned down, and this connects to
- The med checkpoints. Theyre cute, but theyre too good. They contribute like 6 or 7 stasis beds to the map, and have more medkits in each of them. I'd personally suggest making them lower tier beds, like normal ones. The checkpoints themselves arent too egregious but they contribute a lot of power to the station
- The mushroom maint room is really bad for balance. Destroying angel is really rough, even without the null rod added on top. Keep in mind null rods dont just do damage-- they're antimagic too.
- The assistant shrine on the northeast is cute, but a buffed baseball bat and insuls in maints is a bit much, since the bat can act as a reflective vest, if not a budget desword
- The christmas gift next to the meat coffin can have literally any item in the game. This is bad for obvious reasons
- Quantum pads are a bit too optimized for the station in the beginning, without mentioning how some can present a security risk by letting people into secure areas like head offices
- The centcom office is cute for sure, but the problem is the CC clothes within them are super fucking armored. Also the emag is fake for sure, but the corpse underneath it **has a nukie hardsuit, gloves, chameleon ID, and syndie uniform**
- One of the det's offices has normal window panes leading out into the hallway. This is really bad for security
- Ai solars start bolted and arent wired
- You can access the bridge AI upload by deconstructing 1 RWall. Also the ai board spawners in the bridge ai upload are missing some windows

### Notes for Tom :)

- Maint packed mail crate
- Guide to dank mimery in maints
- Russian Armored Coat north of the public garden
- 3 Russian revolvers by my count on the map
- Theres a lot of random cash spawns across the map in various starting offices, ranging from ~200 to 1200 or more credits
- QM has a mining hardsuit roundstart
- Maint anesthetic tank + advanced fire extinguisher north of medbay
- Maint clownborg module (It's in the clown room. You'll see it)
- Survival pod airlocks are used for the assistant shrine in the northeast
- Mining voucher in maints next to the assistant shrine
- Greentext in maints
- Roundstart language manual named "Five" somewhere near the meat coffin
- 2 roundstart alien organ spawners
- Sunflower perma seeds (Basically like giving them nettles)
- Det gear in armory
- Cargo office has... a disposal pipe to dump people into sec. This is objectively fucking hilarious but it could have some problems with deps
- Extra chaplain hats in the relic closet, and the null rod spawns outside it
- Maints bananium and bananium tiles, as well as bananium statues
- It feels like there are a LOT of turret control levers in AI sat
- Roundstart chem inducer
- 10/11 Eva suits, just counting the normal ones
- Departmental sec officers appear to start with more gear than normal, each regarding their department. The med checkpoint, for example, has medkits

**Comment by TheOnlySolitaire** [comment]
I made some edits to points raised by SpaceDragon00 here and John Willard on the Discord. 
A list of changes detailed in the commit.

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 		#include "map_files\HelioStation\HelioStation.dmm"
 		#include "map_files\SeleneStation\SeleneStation.dmm"
 		#include "map_files\tramstation\tramstation.dmm"
+		#include "map_files\tramstation\solitairestation.dmm"
```
Comment (SgtHunk) [inline _maps/_basemap.dm:18]: tramstation\solitairestation

Comment (JohnFulpWillard) [reply _maps/_basemap.dm:18]: S should be capitalized too

Comment (SgtHunk) [reply _maps/_basemap.dm:18]: also should probably go right below selene rather than below tram

File: _maps/solitairestation.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+	"version": 1,
+	"map_name": "Solitaire Station",
+	"map_path": "map_files/SolitaireStation",
+	"map_file": "SolitaireStation.dmm",
```
Comment (SgtHunk) [inline _maps/solitairestation.json:5]:
```suggestion
	"map_file": "Solitairestation.dmm",
```

**Review by SgtHunk — CHANGES_REQUESTED** [review]

**Comment by Helianthus00** [comment]
Here is my map review. It's not all-extensive, but this is what I noticed while combing through it. Using a google doc considering it's like a million images

https://docs.google.com/document/d/1S0HsX0YhoTmnW24raCrKwByWde2N08P1zfJrJauAJyg/edit?usp=sharing

**Comment by Helianthus00** [comment]
Review 2: Mostly pipes and wires this time, plus whatever else I noticed. https://docs.google.com/document/d/1XW4tPMAaddDEOw3hti1A7Pnu2JZS0jtcriFTWoNlg-4/edit?usp=sharing

Also you need to entirely redo the main disposals, since they don't work properly.

**Comment by Helianthus00** [comment]
I'll add some feedback gathered from the TM here:

Looks like the kitchen is still missing things. An oven, in particular, but probably other things too.

**Comment by Helianthus00** [comment]
This is probably fine to merge now

**Pull request merged**